weather.go

```diff
@@ -1,5 +1,138 @@
 package weather
 
-func Get(location, token string) ([]byte, error) {
-	return nil, nil
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type Weather struct {
+	Name    string
```

Comment:
It'll be helpful for clarity to remove all the fields we don't actually use. I think we only use `Weather.Main` and `main.Temp`, don't we?